+ source /root/mugen-riscv/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start to prepare the database config.'
+ message='Start to prepare the database config.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the database config.'
Fri Aug 25 05:50:05 2023 - INFO  - Start to prepare the database config.
+ DNF_INSTALL libreswan
+ pkgs=libreswan
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py install --pkgs libreswan --node 1 --tempfile ''
+ tmpfile2='Last metadata expiration check: 2:05:24 ago on Fri 25 Aug 2023 03:44:42 AM UTC.
Dependencies resolved.
================================================================================
 Package                Architecture   Version                 Repository  Size
================================================================================
Installing:
 libreswan              x86_64         4.11-1.oe2309           OS         1.0 M
Installing dependencies:
 ldns                   x86_64         1.8.3-1.oe2309          OS         162 k
 libreswan-help         x86_64         4.11-1.oe2309           OS         275 k
 nspr                   x86_64         4.35.0-2.oe2309         OS         121 k
 nss                    x86_64         3.89.0-1.oe2309         OS         650 k
 nss-softokn            x86_64         3.89.0-1.oe2309         OS         685 k
 nss-util               x86_64         3.89.0-1.oe2309         OS         464 k

Transaction Summary
================================================================================
Install  7 Packages

Total download size: 3.3 M
Installed size: 13 M
Downloading Packages:
[MIRROR] ldns-1.8.3-1.oe2309.x86_64.rpm: Curl error (7): Couldn'\''t connect to server for http://121.36.84.172/dailybuild/openEuler-23.09/alpha_openeuler-2023-08-14-14-17-53/OS/x86_64/Packages/ldns-1.8.3-1.oe2309.x86_64.rpm [Failed to connect to 121.36.84.172 port 80 after 0 ms: Couldn'\''t connect to server]
[MIRROR] libreswan-4.11-1.oe2309.x86_64.rpm: Curl error (7): Couldn'\''t connect to server for http://121.36.84.172/dailybuild/openEuler-23.09/alpha_openeuler-2023-08-14-14-17-53/OS/x86_64/Packages/libreswan-4.11-1.oe2309.x86_64.rpm [Failed to connect to 121.36.84.172 port 80 after 0 ms: Couldn'\''t connect to server]
[MIRROR] libreswan-help-4.11-1.oe2309.x86_64.rpm: Curl error (7): Couldn'\''t connect to server for http://121.36.84.172/dailybuild/openEuler-23.09/alpha_openeuler-2023-08-14-14-17-53/OS/x86_64/Packages/libreswan-help-4.11-1.oe2309.x86_64.rpm [Failed to connect to 121.36.84.172 port 80 after 0 ms: Couldn'\''t connect to server]
[MIRROR] ldns-1.8.3-1.oe2309.x86_64.rpm: Curl error (7): Couldn'\''t connect to server for http://121.36.84.172/dailybuild/openEuler-23.09/alpha_openeuler-2023-08-14-14-17-53/OS/x86_64/Packages/ldns-1.8.3-1.oe2309.x86_64.rpm [Failed to connect to 121.36.84.172 port 80 after 0 ms: Couldn'\''t connect to server]
[MIRROR] libreswan-4.11-1.oe2309.x86_64.rpm: Curl error (7): Couldn'\''t connect to server for http://121.36.84.172/dailybuild/openEuler-23.09/alpha_openeuler-2023-08-14-14-17-53/OS/x86_64/Packages/libreswan-4.11-1.oe2309.x86_64.rpm [Failed to connect to 121.36.84.172 port 80 after 0 ms: Couldn'\''t connect to server]
[MIRROR] libreswan-help-4.11-1.oe2309.x86_64.rpm: Curl error (7): Couldn'\''t connect to server for http://121.36.84.172/dailybuild/openEuler-23.09/alpha_openeuler-2023-08-14-14-17-53/OS/x86_64/Packages/libreswan-help-4.11-1.oe2309.x86_64.rpm [Failed to connect to 121.36.84.172 port 80 after 0 ms: Couldn'\''t connect to server]
[FAILED] ldns-1.8.3-1.oe2309.x86_64.rpm: No more mirrors to try - All mirrors were already tried without success

The downloaded packages were saved in cache until the next successful transaction.
You can remove cached packages by executing '\''dnf clean packages'\''.
Error: Error downloading packages:
  ldns-1.8.3-1.oe2309.x86_64: Cannot download, all mirrors were already tried without success'
+ '[' -z '' ']'
+ tmpfile='Last metadata expiration check: 2:05:24 ago on Fri 25 Aug 2023 03:44:42 AM UTC.
Dependencies resolved.
================================================================================
 Package                Architecture   Version                 Repository  Size
================================================================================
Installing:
 libreswan              x86_64         4.11-1.oe2309           OS         1.0 M
Installing dependencies:
 ldns                   x86_64         1.8.3-1.oe2309          OS         162 k
 libreswan-help         x86_64         4.11-1.oe2309           OS         275 k
 nspr                   x86_64         4.35.0-2.oe2309         OS         121 k
 nss                    x86_64         3.89.0-1.oe2309         OS         650 k
 nss-softokn            x86_64         3.89.0-1.oe2309         OS         685 k
 nss-util               x86_64         3.89.0-1.oe2309         OS         464 k

Transaction Summary
================================================================================
Install  7 Packages

Total download size: 3.3 M
Installed size: 13 M
Downloading Packages:
[MIRROR] ldns-1.8.3-1.oe2309.x86_64.rpm: Curl error (7): Couldn'\''t connect to server for http://121.36.84.172/dailybuild/openEuler-23.09/alpha_openeuler-2023-08-14-14-17-53/OS/x86_64/Packages/ldns-1.8.3-1.oe2309.x86_64.rpm [Failed to connect to 121.36.84.172 port 80 after 0 ms: Couldn'\''t connect to server]
[MIRROR] libreswan-4.11-1.oe2309.x86_64.rpm: Curl error (7): Couldn'\''t connect to server for http://121.36.84.172/dailybuild/openEuler-23.09/alpha_openeuler-2023-08-14-14-17-53/OS/x86_64/Packages/libreswan-4.11-1.oe2309.x86_64.rpm [Failed to connect to 121.36.84.172 port 80 after 0 ms: Couldn'\''t connect to server]
[MIRROR] libreswan-help-4.11-1.oe2309.x86_64.rpm: Curl error (7): Couldn'\''t connect to server for http://121.36.84.172/dailybuild/openEuler-23.09/alpha_openeuler-2023-08-14-14-17-53/OS/x86_64/Packages/libreswan-help-4.11-1.oe2309.x86_64.rpm [Failed to connect to 121.36.84.172 port 80 after 0 ms: Couldn'\''t connect to server]
[MIRROR] ldns-1.8.3-1.oe2309.x86_64.rpm: Curl error (7): Couldn'\''t connect to server for http://121.36.84.172/dailybuild/openEuler-23.09/alpha_openeuler-2023-08-14-14-17-53/OS/x86_64/Packages/ldns-1.8.3-1.oe2309.x86_64.rpm [Failed to connect to 121.36.84.172 port 80 after 0 ms: Couldn'\''t connect to server]
[MIRROR] libreswan-4.11-1.oe2309.x86_64.rpm: Curl error (7): Couldn'\''t connect to server for http://121.36.84.172/dailybuild/openEuler-23.09/alpha_openeuler-2023-08-14-14-17-53/OS/x86_64/Packages/libreswan-4.11-1.oe2309.x86_64.rpm [Failed to connect to 121.36.84.172 port 80 after 0 ms: Couldn'\''t connect to server]
[MIRROR] libreswan-help-4.11-1.oe2309.x86_64.rpm: Curl error (7): Couldn'\''t connect to server for http://121.36.84.172/dailybuild/openEuler-23.09/alpha_openeuler-2023-08-14-14-17-53/OS/x86_64/Packages/libreswan-help-4.11-1.oe2309.x86_64.rpm [Failed to connect to 121.36.84.172 port 80 after 0 ms: Couldn'\''t connect to server]
[FAILED] ldns-1.8.3-1.oe2309.x86_64.rpm: No more mirrors to try - All mirrors were already tried without success

The downloaded packages were saved in cache until the next successful transaction.
You can remove cached packages by executing '\''dnf clean packages'\''.
Error: Error downloading packages:
  ldns-1.8.3-1.oe2309.x86_64: Cannot download, all mirrors were already tried without success'
+ ipsec restart
oe_test_libreswan_ipsec_algparse.sh: line 25: ipsec: command not found
+ LOG_INFO 'End to prepare the database config.'
+ message='End to prepare the database config.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to prepare the database config.'
Fri Aug 25 05:50:06 2023 - INFO  - End to prepare the database config.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Fri Aug 25 05:50:06 2023 - INFO  - Start to run test.
+ ipsec algparse ike
oe_test_libreswan_ipsec_algparse.sh: line 33: ipsec: command not found
+ CHECK_RESULT 127 0 0 'Check ipsec algparse ike failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check ipsec algparse ike failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check ipsec algparse ike failed.'
+ LOG_ERROR 'Check ipsec algparse ike failed.'
+ message='Check ipsec algparse ike failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ipsec algparse ike failed.'
Fri Aug 25 05:50:06 2023 - ERROR - Check ipsec algparse ike failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_libreswan_ipsec_algparse.sh line 34'
+ message='oe_test_libreswan_ipsec_algparse.sh line 34'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_libreswan_ipsec_algparse.sh line 34'
Fri Aug 25 05:50:06 2023 - ERROR - oe_test_libreswan_ipsec_algparse.sh line 34
+ return 0
+ ipsec algparse esp
oe_test_libreswan_ipsec_algparse.sh: line 35: ipsec: command not found
+ CHECK_RESULT 127 0 0 'Check ipsec algparse esp failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check ipsec algparse esp failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check ipsec algparse esp failed.'
+ LOG_ERROR 'Check ipsec algparse esp failed.'
+ message='Check ipsec algparse esp failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ipsec algparse esp failed.'
Fri Aug 25 05:50:06 2023 - ERROR - Check ipsec algparse esp failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_libreswan_ipsec_algparse.sh line 36'
+ message='oe_test_libreswan_ipsec_algparse.sh line 36'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_libreswan_ipsec_algparse.sh line 36'
Fri Aug 25 05:50:06 2023 - ERROR - oe_test_libreswan_ipsec_algparse.sh line 36
+ return 0
+ ipsec algparse ah
oe_test_libreswan_ipsec_algparse.sh: line 37: ipsec: command not found
+ CHECK_RESULT 127 0 0 'Check ipsec algparse ah failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check ipsec algparse ah failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check ipsec algparse ah failed.'
+ LOG_ERROR 'Check ipsec algparse ah failed.'
+ message='Check ipsec algparse ah failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ipsec algparse ah failed.'
Fri Aug 25 05:50:06 2023 - ERROR - Check ipsec algparse ah failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_libreswan_ipsec_algparse.sh line 38'
+ message='oe_test_libreswan_ipsec_algparse.sh line 38'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_libreswan_ipsec_algparse.sh line 38'
Fri Aug 25 05:50:06 2023 - ERROR - oe_test_libreswan_ipsec_algparse.sh line 38
+ return 0
+ ipsec algparse -tp
oe_test_libreswan_ipsec_algparse.sh: line 39: ipsec: command not found
+ CHECK_RESULT 127 0 0 'Check ipsec algparse -tp failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check ipsec algparse -tp failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check ipsec algparse -tp failed.'
+ LOG_ERROR 'Check ipsec algparse -tp failed.'
+ message='Check ipsec algparse -tp failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ipsec algparse -tp failed.'
Fri Aug 25 05:50:06 2023 - ERROR - Check ipsec algparse -tp failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_libreswan_ipsec_algparse.sh line 40'
+ message='oe_test_libreswan_ipsec_algparse.sh line 40'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_libreswan_ipsec_algparse.sh line 40'
Fri Aug 25 05:50:06 2023 - ERROR - oe_test_libreswan_ipsec_algparse.sh line 40
+ return 0
+ ipsec algparse -ta
oe_test_libreswan_ipsec_algparse.sh: line 41: ipsec: command not found
+ CHECK_RESULT 127 0 0 'Check ipsec algparse -ta failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check ipsec algparse -ta failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check ipsec algparse -ta failed.'
+ LOG_ERROR 'Check ipsec algparse -ta failed.'
+ message='Check ipsec algparse -ta failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ipsec algparse -ta failed.'
Fri Aug 25 05:50:06 2023 - ERROR - Check ipsec algparse -ta failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_libreswan_ipsec_algparse.sh line 42'
+ message='oe_test_libreswan_ipsec_algparse.sh line 42'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_libreswan_ipsec_algparse.sh line 42'
Fri Aug 25 05:50:06 2023 - ERROR - oe_test_libreswan_ipsec_algparse.sh line 42
+ return 0
+ ipsec algparse -v2 'ike=aes_gcm-none-sha2;modp2048'
oe_test_libreswan_ipsec_algparse.sh: line 43: ipsec: command not found
+ CHECK_RESULT 127 0 0 'Check ipsec algparse  -v2 '\''ike=aes_gcm-none-sha2;modp2048'\'' failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check ipsec algparse  -v2 '\''ike=aes_gcm-none-sha2;modp2048'\'' failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check ipsec algparse  -v2 '\''ike=aes_gcm-none-sha2;modp2048'\'' failed.'
+ LOG_ERROR 'Check ipsec algparse  -v2 '\''ike=aes_gcm-none-sha2;modp2048'\'' failed.'
+ message='Check ipsec algparse  -v2 '\''ike=aes_gcm-none-sha2;modp2048'\'' failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ipsec algparse  -v2 '\''ike=aes_gcm-none-sha2;modp2048'\'' failed.'
Fri Aug 25 05:50:06 2023 - ERROR - Check ipsec algparse  -v2 'ike=aes_gcm-none-sha2;modp2048' failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_libreswan_ipsec_algparse.sh line 44'
+ message='oe_test_libreswan_ipsec_algparse.sh line 44'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_libreswan_ipsec_algparse.sh line 44'
Fri Aug 25 05:50:06 2023 - ERROR - oe_test_libreswan_ipsec_algparse.sh line 44
+ return 0
+ ipsec algparse -v1
oe_test_libreswan_ipsec_algparse.sh: line 45: ipsec: command not found
+ CHECK_RESULT 127 0 0 'Check ipsec algparse -v1 failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check ipsec algparse -v1 failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check ipsec algparse -v1 failed.'
+ LOG_ERROR 'Check ipsec algparse -v1 failed.'
+ message='Check ipsec algparse -v1 failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ipsec algparse -v1 failed.'
Fri Aug 25 05:50:06 2023 - ERROR - Check ipsec algparse -v1 failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_libreswan_ipsec_algparse.sh line 46'
+ message='oe_test_libreswan_ipsec_algparse.sh line 46'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_libreswan_ipsec_algparse.sh line 46'
Fri Aug 25 05:50:06 2023 - ERROR - oe_test_libreswan_ipsec_algparse.sh line 46
+ return 0
+ ipsec algparse -pfs
oe_test_libreswan_ipsec_algparse.sh: line 47: ipsec: command not found
+ CHECK_RESULT 127 0 0 'Check ipsec algparse -pfs failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check ipsec algparse -pfs failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check ipsec algparse -pfs failed.'
+ LOG_ERROR 'Check ipsec algparse -pfs failed.'
+ message='Check ipsec algparse -pfs failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ipsec algparse -pfs failed.'
Fri Aug 25 05:50:06 2023 - ERROR - Check ipsec algparse -pfs failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_libreswan_ipsec_algparse.sh line 48'
+ message='oe_test_libreswan_ipsec_algparse.sh line 48'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_libreswan_ipsec_algparse.sh line 48'
Fri Aug 25 05:50:06 2023 - ERROR - oe_test_libreswan_ipsec_algparse.sh line 48
+ return 0
+ ipsec algparse -P testpwd
oe_test_libreswan_ipsec_algparse.sh: line 49: ipsec: command not found
+ CHECK_RESULT 127 0 0 'Check ipsec algparse -P testpwd failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check ipsec algparse -P testpwd failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check ipsec algparse -P testpwd failed.'
+ LOG_ERROR 'Check ipsec algparse -P testpwd failed.'
+ message='Check ipsec algparse -P testpwd failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ipsec algparse -P testpwd failed.'
Fri Aug 25 05:50:06 2023 - ERROR - Check ipsec algparse -P testpwd failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_libreswan_ipsec_algparse.sh line 50'
+ message='oe_test_libreswan_ipsec_algparse.sh line 50'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_libreswan_ipsec_algparse.sh line 50'
Fri Aug 25 05:50:06 2023 - ERROR - oe_test_libreswan_ipsec_algparse.sh line 50
+ return 0
+ ipsec algparse -v
oe_test_libreswan_ipsec_algparse.sh: line 51: ipsec: command not found
+ CHECK_RESULT 127 0 0 'Check ipsec algparse -v failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check ipsec algparse -v failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check ipsec algparse -v failed.'
+ LOG_ERROR 'Check ipsec algparse -v failed.'
+ message='Check ipsec algparse -v failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ipsec algparse -v failed.'
Fri Aug 25 05:50:06 2023 - ERROR - Check ipsec algparse -v failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_libreswan_ipsec_algparse.sh line 52'
+ message='oe_test_libreswan_ipsec_algparse.sh line 52'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_libreswan_ipsec_algparse.sh line 52'
Fri Aug 25 05:50:06 2023 - ERROR - oe_test_libreswan_ipsec_algparse.sh line 52
+ return 0
+ ipsec algparse --verbose
oe_test_libreswan_ipsec_algparse.sh: line 53: ipsec: command not found
+ CHECK_RESULT 127 0 0 'Check ipsec algparse --verbose failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check ipsec algparse --verbose failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check ipsec algparse --verbose failed.'
+ LOG_ERROR 'Check ipsec algparse --verbose failed.'
+ message='Check ipsec algparse --verbose failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ipsec algparse --verbose failed.'
Fri Aug 25 05:50:07 2023 - ERROR - Check ipsec algparse --verbose failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_libreswan_ipsec_algparse.sh line 54'
+ message='oe_test_libreswan_ipsec_algparse.sh line 54'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_libreswan_ipsec_algparse.sh line 54'
Fri Aug 25 05:50:07 2023 - ERROR - oe_test_libreswan_ipsec_algparse.sh line 54
+ return 0
+ ipsec algparse --debug
oe_test_libreswan_ipsec_algparse.sh: line 55: ipsec: command not found
+ CHECK_RESULT 127 0 0 'Check ipsec algparse --debug failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check ipsec algparse --debug failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check ipsec algparse --debug failed.'
+ LOG_ERROR 'Check ipsec algparse --debug failed.'
+ message='Check ipsec algparse --debug failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ipsec algparse --debug failed.'
Fri Aug 25 05:50:07 2023 - ERROR - Check ipsec algparse --debug failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_libreswan_ipsec_algparse.sh line 56'
+ message='oe_test_libreswan_ipsec_algparse.sh line 56'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_libreswan_ipsec_algparse.sh line 56'
Fri Aug 25 05:50:07 2023 - ERROR - oe_test_libreswan_ipsec_algparse.sh line 56
+ return 0
+ ipsec algparse --impair
oe_test_libreswan_ipsec_algparse.sh: line 57: ipsec: command not found
+ CHECK_RESULT 127 0 0 'Check ipsec algparse --impair failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check ipsec algparse --impair failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check ipsec algparse --impair failed.'
+ LOG_ERROR 'Check ipsec algparse --impair failed.'
+ message='Check ipsec algparse --impair failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ipsec algparse --impair failed.'
Fri Aug 25 05:50:07 2023 - ERROR - Check ipsec algparse --impair failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_libreswan_ipsec_algparse.sh line 58'
+ message='oe_test_libreswan_ipsec_algparse.sh line 58'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_libreswan_ipsec_algparse.sh line 58'
Fri Aug 25 05:50:07 2023 - ERROR - oe_test_libreswan_ipsec_algparse.sh line 58
+ return 0
+ ipsec algparse --ignore
oe_test_libreswan_ipsec_algparse.sh: line 59: ipsec: command not found
+ CHECK_RESULT 127 0 0 'Check ipsec algparse --ignore failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check ipsec algparse --ignore failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check ipsec algparse --ignore failed.'
+ LOG_ERROR 'Check ipsec algparse --ignore failed.'
+ message='Check ipsec algparse --ignore failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ipsec algparse --ignore failed.'
Fri Aug 25 05:50:07 2023 - ERROR - Check ipsec algparse --ignore failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_libreswan_ipsec_algparse.sh line 60'
+ message='oe_test_libreswan_ipsec_algparse.sh line 60'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_libreswan_ipsec_algparse.sh line 60'
Fri Aug 25 05:50:07 2023 - ERROR - oe_test_libreswan_ipsec_algparse.sh line 60
+ return 0
+ ipsec algparse -p1
oe_test_libreswan_ipsec_algparse.sh: line 61: ipsec: command not found
+ CHECK_RESULT 127 0 0 'Check ipsec algparse -p1 failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check ipsec algparse -p1 failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check ipsec algparse -p1 failed.'
+ LOG_ERROR 'Check ipsec algparse -p1 failed.'
+ message='Check ipsec algparse -p1 failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ipsec algparse -p1 failed.'
Fri Aug 25 05:50:07 2023 - ERROR - Check ipsec algparse -p1 failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_libreswan_ipsec_algparse.sh line 62'
+ message='oe_test_libreswan_ipsec_algparse.sh line 62'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_libreswan_ipsec_algparse.sh line 62'
Fri Aug 25 05:50:07 2023 - ERROR - oe_test_libreswan_ipsec_algparse.sh line 62
+ return 0
+ ipsec algparse -p2
oe_test_libreswan_ipsec_algparse.sh: line 63: ipsec: command not found
+ CHECK_RESULT 127 0 0 'Check ipsec algparse -p2 failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check ipsec algparse -p2 failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check ipsec algparse -p2 failed.'
+ LOG_ERROR 'Check ipsec algparse -p2 failed.'
+ message='Check ipsec algparse -p2 failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ipsec algparse -p2 failed.'
Fri Aug 25 05:50:07 2023 - ERROR - Check ipsec algparse -p2 failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_libreswan_ipsec_algparse.sh line 64'
+ message='oe_test_libreswan_ipsec_algparse.sh line 64'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_libreswan_ipsec_algparse.sh line 64'
Fri Aug 25 05:50:07 2023 - ERROR - oe_test_libreswan_ipsec_algparse.sh line 64
+ return 0
+ ipsec algparse -h
oe_test_libreswan_ipsec_algparse.sh: line 65: ipsec: command not found
+ CHECK_RESULT 127 0 0 'Check ipsec algparse -h failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check ipsec algparse -h failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check ipsec algparse -h failed.'
+ LOG_ERROR 'Check ipsec algparse -h failed.'
+ message='Check ipsec algparse -h failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ipsec algparse -h failed.'
Fri Aug 25 05:50:07 2023 - ERROR - Check ipsec algparse -h failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_libreswan_ipsec_algparse.sh line 66'
+ message='oe_test_libreswan_ipsec_algparse.sh line 66'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_libreswan_ipsec_algparse.sh line 66'
Fri Aug 25 05:50:07 2023 - ERROR - oe_test_libreswan_ipsec_algparse.sh line 66
+ return 0
+ ipsec algparse -h
+ CHECK_RESULT 127 1 0 'Check ipsec algparse -h -d deleted failed.'
+ actual_result=127
+ expect_result=1
+ mode=0
+ error_log='Check ipsec algparse -h -d deleted failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 1x
+ test -n 'Check ipsec algparse -h -d deleted failed.'
+ LOG_ERROR 'Check ipsec algparse -h -d deleted failed.'
+ message='Check ipsec algparse -h -d deleted failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ipsec algparse -h -d deleted failed.'
Fri Aug 25 05:50:07 2023 - ERROR - Check ipsec algparse -h -d deleted failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_libreswan_ipsec_algparse.sh line 70'
+ message='oe_test_libreswan_ipsec_algparse.sh line 70'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_libreswan_ipsec_algparse.sh line 70'
Fri Aug 25 05:50:07 2023 - ERROR - oe_test_libreswan_ipsec_algparse.sh line 70
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Fri Aug 25 05:50:07 2023 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 18
+ test 18 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Fri Aug 25 05:50:07 2023 - ERROR - failed to execute the case.
+ exit 18
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Fri Aug 25 05:50:07 2023 - INFO  - Start to restore the test environment.
+ rm -f testlog
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z Last metadata expiration check: 2:05:24 ago on Fri 25 Aug 2023 03:44:42 AM UTC.
Dependencies resolved.
================================================================================
 Package                Architecture   Version                 Repository  Size
================================================================================
Installing:
 libreswan              x86_64         4.11-1.oe2309           OS         1.0 M
Installing dependencies:
 ldns                   x86_64         1.8.3-1.oe2309          OS         162 k
 libreswan-help         x86_64         4.11-1.oe2309           OS         275 k
 nspr                   x86_64         4.35.0-2.oe2309         OS         121 k
 nss                    x86_64         3.89.0-1.oe2309         OS         650 k
 nss-softokn            x86_64         3.89.0-1.oe2309         OS         685 k
 nss-util               x86_64         3.89.0-1.oe2309         OS         464 k

Transaction Summary
================================================================================
Install  7 Packages

Total download size: 3.3 M
Installed size: 13 M
Downloading Packages:
[MIRROR] ldns-1.8.3-1.oe2309.x86_64.rpm: Curl error (7): Couldn't connect to server for http://121.36.84.172/dailybuild/openEuler-23.09/alpha_openeuler-2023-08-14-14-17-53/OS/x86_64/Packages/ldns-1.8.3-1.oe2309.x86_64.rpm [Failed to connect to 121.36.84.172 port 80 after 0 ms: Couldn't connect to server]
[MIRROR] libreswan-4.11-1.oe2309.x86_64.rpm: Curl error (7): Couldn't connect to server for http://121.36.84.172/dailybuild/openEuler-23.09/alpha_openeuler-2023-08-14-14-17-53/OS/x86_64/Packages/libreswan-4.11-1.oe2309.x86_64.rpm [Failed to connect to 121.36.84.172 port 80 after 0 ms: Couldn't connect to server]
[MIRROR] libreswan-help-4.11-1.oe2309.x86_64.rpm: Curl error (7): Couldn't connect to server for http://121.36.84.172/dailybuild/openEuler-23.09/alpha_openeuler-2023-08-14-14-17-53/OS/x86_64/Packages/libreswan-help-4.11-1.oe2309.x86_64.rpm [Failed to connect to 121.36.84.172 port 80 after 0 ms: Couldn't connect to server]
[MIRROR] ldns-1.8.3-1.oe2309.x86_64.rpm: Curl error (7): Couldn't connect to server for http://121.36.84.172/dailybuild/openEuler-23.09/alpha_openeuler-2023-08-14-14-17-53/OS/x86_64/Packages/ldns-1.8.3-1.oe2309.x86_64.rpm [Failed to connect to 121.36.84.172 port 80 after 0 ms: Couldn't connect to server]
[MIRROR] libreswan-4.11-1.oe2309.x86_64.rpm: Curl error (7): Couldn't connect to server for http://121.36.84.172/dailybuild/openEuler-23.09/alpha_openeuler-2023-08-14-14-17-53/OS/x86_64/Packages/libreswan-4.11-1.oe2309.x86_64.rpm [Failed to connect to 121.36.84.172 port 80 after 0 ms: Couldn't connect to server]
[MIRROR] libreswan-help-4.11-1.oe2309.x86_64.rpm: Curl error (7): Couldn't connect to server for http://121.36.84.172/dailybuild/openEuler-23.09/alpha_openeuler-2023-08-14-14-17-53/OS/x86_64/Packages/libreswan-help-4.11-1.oe2309.x86_64.rpm [Failed to connect to 121.36.84.172 port 80 after 0 ms: Couldn't connect to server]
[FAILED] ldns-1.8.3-1.oe2309.x86_64.rpm: No more mirrors to try - All mirrors were already tried without success

The downloaded packages were saved in cache until the next successful transaction.
You can remove cached packages by executing 'dnf clean packages'.
Error: Error downloading packages:
  ldns-1.8.3-1.oe2309.x86_64: Cannot download, all mirrors were already tried without success ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile 'Last metadata expiration check: 2:05:24 ago on Fri 25 Aug 2023 03:44:42 AM UTC.
Dependencies resolved.
================================================================================
 Package                Architecture   Version                 Repository  Size
================================================================================
Installing:
 libreswan              x86_64         4.11-1.oe2309           OS         1.0 M
Installing dependencies:
 ldns                   x86_64         1.8.3-1.oe2309          OS         162 k
 libreswan-help         x86_64         4.11-1.oe2309           OS         275 k
 nspr                   x86_64         4.35.0-2.oe2309         OS         121 k
 nss                    x86_64         3.89.0-1.oe2309         OS         650 k
 nss-softokn            x86_64         3.89.0-1.oe2309         OS         685 k
 nss-util               x86_64         3.89.0-1.oe2309         OS         464 k

Transaction Summary
================================================================================
Install  7 Packages

Total download size: 3.3 M
Installed size: 13 M
Downloading Packages:
[MIRROR] ldns-1.8.3-1.oe2309.x86_64.rpm: Curl error (7): Couldn'\''t connect to server for http://121.36.84.172/dailybuild/openEuler-23.09/alpha_openeuler-2023-08-14-14-17-53/OS/x86_64/Packages/ldns-1.8.3-1.oe2309.x86_64.rpm [Failed to connect to 121.36.84.172 port 80 after 0 ms: Couldn'\''t connect to server]
[MIRROR] libreswan-4.11-1.oe2309.x86_64.rpm: Curl error (7): Couldn'\''t connect to server for http://121.36.84.172/dailybuild/openEuler-23.09/alpha_openeuler-2023-08-14-14-17-53/OS/x86_64/Packages/libreswan-4.11-1.oe2309.x86_64.rpm [Failed to connect to 121.36.84.172 port 80 after 0 ms: Couldn'\''t connect to server]
[MIRROR] libreswan-help-4.11-1.oe2309.x86_64.rpm: Curl error (7): Couldn'\''t connect to server for http://121.36.84.172/dailybuild/openEuler-23.09/alpha_openeuler-2023-08-14-14-17-53/OS/x86_64/Packages/libreswan-help-4.11-1.oe2309.x86_64.rpm [Failed to connect to 121.36.84.172 port 80 after 0 ms: Couldn'\''t connect to server]
[MIRROR] ldns-1.8.3-1.oe2309.x86_64.rpm: Curl error (7): Couldn'\''t connect to server for http://121.36.84.172/dailybuild/openEuler-23.09/alpha_openeuler-2023-08-14-14-17-53/OS/x86_64/Packages/ldns-1.8.3-1.oe2309.x86_64.rpm [Failed to connect to 121.36.84.172 port 80 after 0 ms: Couldn'\''t connect to server]
[MIRROR] libreswan-4.11-1.oe2309.x86_64.rpm: Curl error (7): Couldn'\''t connect to server for http://121.36.84.172/dailybuild/openEuler-23.09/alpha_openeuler-2023-08-14-14-17-53/OS/x86_64/Packages/libreswan-4.11-1.oe2309.x86_64.rpm [Failed to connect to 121.36.84.172 port 80 after 0 ms: Couldn'\''t connect to server]
[MIRROR] libreswan-help-4.11-1.oe2309.x86_64.rpm: Curl error (7): Couldn'\''t connect to server for http://121.36.84.172/dailybuild/openEuler-23.09/alpha_openeuler-2023-08-14-14-17-53/OS/x86_64/Packages/libreswan-help-4.11-1.oe2309.x86_64.rpm [Failed to connect to 121.36.84.172 port 80 after 0 ms: Couldn'\''t connect to server]
[FAILED] ldns-1.8.3-1.oe2309.x86_64.rpm: No more mirrors to try - All mirrors were already tried without success

The downloaded packages were saved in cache until the next successful transaction.
You can remove cached packages by executing '\''dnf clean packages'\''.
Error: Error downloading packages:
  ldns-1.8.3-1.oe2309.x86_64: Cannot download, all mirrors were already tried without success'
Traceback (most recent call last):
  File "/root/mugen-riscv/libs/locallibs/rpm_manage.py", line 181, in <module>
    exitcode = rpm_remove(args.node, args.pkgs, args.tempfile)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/mugen-riscv/libs/locallibs/rpm_manage.py", line 152, in rpm_remove
    with open(tmpfile, "r") as f:
         ^^^^^^^^^^^^^^^^^^
OSError: [Errno 36] File name too long: "Last metadata expiration check: 2:05:24 ago on Fri 25 Aug 2023 03:44:42 AM UTC.\nDependencies resolved.\n================================================================================\n Package                Architecture   Version                 Repository  Size\n================================================================================\nInstalling:\n libreswan              x86_64         4.11-1.oe2309           OS         1.0 M\nInstalling dependencies:\n ldns                   x86_64         1.8.3-1.oe2309          OS         162 k\n libreswan-help         x86_64         4.11-1.oe2309           OS         275 k\n nspr                   x86_64         4.35.0-2.oe2309         OS         121 k\n nss                    x86_64         3.89.0-1.oe2309         OS         650 k\n nss-softokn            x86_64         3.89.0-1.oe2309         OS         685 k\n nss-util               x86_64         3.89.0-1.oe2309         OS         464 k\n\nTransaction Summary\n================================================================================\nInstall  7 Packages\n\nTotal download size: 3.3 M\nInstalled size: 13 M\nDownloading Packages:\n[MIRROR] ldns-1.8.3-1.oe2309.x86_64.rpm: Curl error (7): Couldn't connect to server for http://121.36.84.172/dailybuild/openEuler-23.09/alpha_openeuler-2023-08-14-14-17-53/OS/x86_64/Packages/ldns-1.8.3-1.oe2309.x86_64.rpm [Failed to connect to 121.36.84.172 port 80 after 0 ms: Couldn't connect to server]\n[MIRROR] libreswan-4.11-1.oe2309.x86_64.rpm: Curl error (7): Couldn't connect to server for http://121.36.84.172/dailybuild/openEuler-23.09/alpha_openeuler-2023-08-14-14-17-53/OS/x86_64/Packages/libreswan-4.11-1.oe2309.x86_64.rpm [Failed to connect to 121.36.84.172 port 80 after 0 ms: Couldn't connect to server]\n[MIRROR] libreswan-help-4.11-1.oe2309.x86_64.rpm: Curl error (7): Couldn't connect to server for http://121.36.84.172/dailybuild/openEuler-23.09/alpha_openeuler-2023-08-14-14-17-53/OS/x86_64/Packages/libreswan-help-4.11-1.oe2309.x86_64.rpm [Failed to connect to 121.36.84.172 port 80 after 0 ms: Couldn't connect to server]\n[MIRROR] ldns-1.8.3-1.oe2309.x86_64.rpm: Curl error (7): Couldn't connect to server for http://121.36.84.172/dailybuild/openEuler-23.09/alpha_openeuler-2023-08-14-14-17-53/OS/x86_64/Packages/ldns-1.8.3-1.oe2309.x86_64.rpm [Failed to connect to 121.36.84.172 port 80 after 0 ms: Couldn't connect to server]\n[MIRROR] libreswan-4.11-1.oe2309.x86_64.rpm: Curl error (7): Couldn't connect to server for http://121.36.84.172/dailybuild/openEuler-23.09/alpha_openeuler-2023-08-14-14-17-53/OS/x86_64/Packages/libreswan-4.11-1.oe2309.x86_64.rpm [Failed to connect to 121.36.84.172 port 80 after 0 ms: Couldn't connect to server]\n[MIRROR] libreswan-help-4.11-1.oe2309.x86_64.rpm: Curl error (7): Couldn't connect to server for http://121.36.84.172/dailybuild/openEuler-23.09/alpha_openeuler-2023-08-14-14-17-53/OS/x86_64/Packages/libreswan-help-4.11-1.oe2309.x86_64.rpm [Failed to connect to 121.36.84.172 port 80 after 0 ms: Couldn't connect to server]\n[FAILED] ldns-1.8.3-1.oe2309.x86_64.rpm: No more mirrors to try - All mirrors were already tried without success\n\nThe downloaded packages were saved in cache until the next successful transaction.\nYou can remove cached packages by executing 'dnf clean packages'.\nError: Error downloading packages:\n  ldns-1.8.3-1.oe2309.x86_64: Cannot download, all mirrors were already tried without success"
+ '[' 0 -ne 0 ']'
+ LOG_INFO 'End to restore the test environment.'
+ message='End to restore the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Fri Aug 25 05:50:07 2023 - INFO  - End to restore the test environment.
